packages/components/src/components/dropdown/dropdown.stories.ts

```diff
@@ -101,260 +115,149 @@ export const Default = {
 };
 
 /**
- * For round triggers (like e. g. default `sd-buttons`), set the `rounded` attribute. This sets a rounding on the dropdown and automatically adds a minimal distance of 1px between the trigger and the panel.
+ * Use the `open` attribute to toggle the visibility of the dropdown.
+ */
+
+export const Open = {
+  name: 'Open',
+  render: () => html`
+    <sd-dropdown open>
+      <div class="slot slot--border slot--background w-52 h-12">Default Slot</div>
+      <sd-button slot="trigger">Trigger</sd-button>
+    </sd-dropdown>
+  `
+};
+
+/**
+ * Use the `rounded` attribute to set a border-radius on the dropdown panel and trigger button.
  */
 
 export const Rounded = {
-  parameters: { controls: { exclude: ['rounded', 'default'] } },
-  render: (args: any) =>
-    generateTemplate({
-      axis: {
-        x: {
-          type: 'attribute',
-          name: 'rounded'
-        }
-      },
-      constants: {
-        type: 'slot',
-        name: 'default',
-        value: '<div class="example"></div>'
-      },
-      args
-    })
+  name: 'Rounded',
+  render: () => html`
+    <sd-dropdown open rounded>
+      <div class="slot slot--border slot--background w-52 h-12">Default Slot</div>
+      <sd-button slot="trigger">Trigger</sd-button>
+    </sd-dropdown>
+  `
 };
 
 /**
- * The dropdown automatically adjusts its size to fit the screen. The content gets scrollable then. To disable this behavior, set the `no-auto-size` attribute.
- * Note: `no-flip` is activated here.
+ * Use the `placement` attribute to set the position of the dropdown panel relative to the trigger.
  */
 
-export const NoAutoSize = {
-  parameters: { controls: { exclude: ['default', 'no-flip', 'no-auto-size'] } },
-  render: (args: any) =>
-    generateTemplate({
-      axis: {
-        x: {
-          type: 'attribute',
-          name: 'no-auto-size'
-        }
-      },
-      constants: [
-        {
-          type: 'slot',
-          name: 'default',
-          value:
-            '<div style="width: 120px;"><div style="height: 110vh; padding: 12px;"><div  class="slot slot--border slot--text slot--background" style="height: 100%; width: auto;">Scroll down<br><br>⬇️</div></div></div>'
-        },
-        {
-          type: 'attribute',
-          name: 'no-flip',
-          value: true
-        }
-      ],
-      args
-    })
+export const Placement = {
+  name: 'Placement',
```

Comment:
I would only show 2 placements

https://www.figma.com/design/YDktJcseQIIQbsuCpoKS4V/Component-Docs?node-id=2357-10943&t=UpPGX0jehyNFfgNq-11

Reply:
In figma you are showing placements `bottom-start` and `bottom-end` but in the trigger it's written `top-start` and `top-end`. I adjusted to `bottom-*`on the trigger and placement.